package.json

```diff
@@ -29,8 +29,20 @@
     "build": "netlify-lambda build src -c webpack.functions.js",
     "format": "prettier --write .",
     "precommit": "yarn format && yarn test",
-    "reset-db": "node scripts/create-database && node scripts/import-spreadsheet.js",
     "los": "node scripts/los.js",
+    "migrate": "node-pg-migrate --reject-unauthorized false",
```

Comment:
We should eventually configure ssl properly! https://node-postgres.com/features/ssl